tests/react/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "react",
```

Comment:
Why a `package.json` here and not in the `base` folder?

Reply:
This is used to fake the presence of dependencies so that ESLint does not generate errors of the type:

> 'react' should be listed in the project's dependencies

Also we haven't to add this dependencies to the root `package.json` file.

Reply:
I've just removed all useless attributes.